jitify2.hpp

```diff
@@ -53,9 +53,9 @@
 #include <unordered_map>
 #include <vector>
 
-#if __cplusplus >= 201703L
-#include <string_view>
-#endif
+// #if __cplusplus >= 201703L
+// #include <string_view>
+// #endif
```

Comment:
Why do you comment out those lines? Are they going back sometimes in the future? Is there any better way to check for completeness of `string_view`?

Reply:
this is just the fastest way around fixing the fact that `std::string_view` isn't compatible with `std::basic_string<char>` when calling `linker_options->push_back(key);` There's no real need to use `string_view` here. These changes are to get people unblocked for building with c++17, but they're not necessarily the "best" solution.